src/doc/rustc/src/platform-support/wasm32-wasi-preview1-threads.md

```diff
@@ -0,0 +1,151 @@
+# `wasm32-wasi-preview1-threads`
+
+**Tier: 3**
```

Comment:
I think this should be tier 2 right? 🙂

Reply:
It would be great if this was tier 2. However, my current understanding is that [the policy ](https://doc.rust-lang.org/rustc/target-tier-policy.html) suggests to put this in Tier3 at first and then move it up a tier.